XiEditor/EditView.swift

```diff
@@ -71,22 +71,45 @@ func colorFromArgb(_ argb: UInt32) -> NSColor {
         alpha: CGFloat((argb >> 24) & 0xff) * 1.0/255)
 }
 
-func camelCaseToUnderscored(_ name: NSString) -> NSString {
-    let underscored = NSMutableString();
-    let scanner = Scanner(string: name as String);
-    let notUpperCase = CharacterSet.uppercaseLetters.inverted;
-    var notUpperCaseFragment: NSString?
-    while (scanner.scanUpToCharacters(from: CharacterSet.uppercaseLetters, into: &notUpperCaseFragment)) {
-        underscored.append(notUpperCaseFragment! as String);
-        var upperCaseFragement: NSString?
-        if (scanner.scanUpToCharacters(from: notUpperCase, into: &upperCaseFragement)) {
-            underscored.append("_");
-            let downcasedFragment = upperCaseFragement!.lowercased;
-            underscored.append(downcasedFragment);
-        }
-    }
-    return underscored;
-}
+// Mapping of selectors to simple no-parameter commands.
```

Comment:
Nit: I might make this a static var in `EditView`, but that might be a hangover from a time when swiftc had performance issues around globals. If a global, I would give it an uppercase name?